### DISCUSSION TITLE
logo svg

### DISCUSSION THREAD

**Discussion by Mick4801** [bootscore/bootscore#516] [Q&A]
Hello
i have a problem with my logo svg, thath does not appears online?
[https://pijamino.com](url)

thanks

**Comment by crftwrk** [comment]
Hello,

svg was not in viewport. At best, grab the original bootScore logo and replace it with your logo in a svg editor. Here is the correct one.

![logo](https://github.com/bootscore/bootscore/assets/51531217/b8494467-a851-463d-9ca2-d5c56350c5f0)

- right click on the image and save
- rename to `logo.svg`
- upload to your folder
- clear cache
- reload

Solved?

**Reply by Mick4801** [reply]
Hi
great, now 's working.
Many thanks.

**Reply by crftwrk** [reply]
You're welcome, please mark answer as solved.
